File: articles/feedback-loops-in-algorithmic-systems/prolog/feedback_risk.pl
feedback_risk(Amplification, Concentration, Intervention, Drift, RecursiveData, Score) :-
    Score is (Amplification + Concentration + Intervention + Drift + RecursiveData) / 5.

% Query example:
% ?- feedback_risk(0.82, 0.76, 0.44, 0.28, 0.31, Score).

File: articles/feedback-loops-in-algorithmic-systems/haskell/FeedbackRisk.hs
module Main where

main :: IO ()
main = do
  let amplification = 0.82
      concentration = 0.76
      intervention = 0.44
      drift = 0.28
      recursiveData = 0.31
      score = (amplification + concentration + intervention + drift + recursiveData) / 5
  putStrLn ("feedback_risk_score=" ++ show score)
